unikraft/export/v0/ukrandom/params.go

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2024, Unikraft GmbH and The KraftKit Authors.
+// Licensed under the BSD-3-Clause License (the "License").
+// You may not use this file except in compliance with the License.
+package ukrandom
+
+import (
+	"crypto/rand"
+	"fmt"
+	"strings"
+
+	"kraftkit.sh/unikraft/export/v0/ukargparse"
+)
+
+var ParamRandomBytes = ukargparse.NewParamStrSlice("random", "seed", nil)
```

Comment:
Let's call this var, to reflect its name in the cmdline, just `ParamRandomSeed`.

Subsequent changes are then necessary to the constructor.